dpnp/dpnp_iface_arraycreation.py

```diff
@@ -1369,64 +1372,22 @@ def meshgrid(*xi, copy=True, sparse=False, indexing="xy"):
     return call_origin(numpy.meshgrid, xi, copy, sparse, indexing)
 
 
-class MGridClass:
-    """
-    Construct a dense multi-dimensional "meshgrid".
-
-    For full documentation refer to :obj:`numpy.mgrid`.
-
-    Examples
-    --------
-    >>> import dpnp as np
-    >>> np.mgrid[0:5,0:5]
-    array([[[0, 0, 0, 0, 0],
-            [1, 1, 1, 1, 1],
-            [2, 2, 2, 2, 2],
-            [3, 3, 3, 3, 3],
-            [4, 4, 4, 4, 4]],
-           [[0, 1, 2, 3, 4],
-            [0, 1, 2, 3, 4],
-            [0, 1, 2, 3, 4],
-            [0, 1, 2, 3, 4],
-            [0, 1, 2, 3, 4]]])
-    >>> np.mgrid[-1:1:5j]
-    array([-1. , -0.5,  0. ,  0.5,  1. ])
-
-    """
-
-    def __getitem__(self, key):
-        return dpnp.array(numpy.mgrid[key])
-
-
-mgrid = MGridClass()
-
-
-class OGridClass:
-    """
-    Construct an open multi-dimensional "meshgrid".
+def mgrid_device(*, device=None, usm_type="device", sycl_queue=None):
```

Comment:
Can we implement `MGridClass` with `__call__` method instead of separate methods with device keywords?